### ISSUE TITLE
Unable to create a gauge-widget on a newrelic dashboard

### ISSUE THREAD

**Issue by BassplayerPatrick** [terraform-providers/terraform-provider-newrelic#32]
Hi,

I'm using terraform version v0.11.7 with provider.newrelic v1.0.0

when applying the following code-snippet, it results in an Internal Server Error
`  widget {`
`    title         = "Pending messages"`
`    row           = 1`
`    column        = 2`
`    visualization = "gauge"`
`    nrql          = "SELECT latest(pending_message_count) FROM message_server"`
`  }`

If i change gauge to billboard or billboard-comparison, it works.

There is also no attribute to set the gauge-limit.

Please fix

**Comment by ctrombley** [comment]
This is blocked on merging https://github.com/paultyng/go-newrelic/pull/71.

**Comment by ctrombley** [comment]
Duplicate of #194

**Comment by ctrombley** [comment]
Will PR this once paultyng/go-newrelic#71 is included in a release.

**Comment by ctrombley** [comment]
Addressed in #206
